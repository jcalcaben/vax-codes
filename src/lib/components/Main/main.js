import React from "react";

import { root, container } from "./main.module.css";

const Main = (props) => {
  const { children } = props;

  return (
    <main className={root}>
      <div className={container}>{children}</div>
    </main>
  );
};

export default Main;
