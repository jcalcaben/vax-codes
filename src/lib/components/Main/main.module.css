.root {
	composes: grid from global;
}

.container {
	composes: w-5/6 from global;
	composes: justify-self-center from global;

}