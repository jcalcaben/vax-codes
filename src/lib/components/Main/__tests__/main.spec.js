import React from 'react';
import Main from '../main';
import ShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ShallowRenderer();

describe("Main component", () => {
  test("renders the correct basic shape", () => {
    renderer.render(<Main>Content</Main>);
		const output = renderer.getRenderOutput();

    expect(output).toMatchSnapshot();
  });
});
