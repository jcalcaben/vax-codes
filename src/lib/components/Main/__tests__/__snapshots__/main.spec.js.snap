// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component renders the correct basic shape 1`] = `
<main>
  Content
</main>
`;
